linera-core/src/client/signer.rs

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::BTreeMap;
+
+use linera_base::{
+    crypto::{AccountPublicKey, AccountSecretKey, AccountSignature, CryptoHash, CryptoRng, Signer},
+    identifiers::AccountOwner,
+};
+use serde::{Deserialize, Serialize};
+
+/// In-memory signer.
+pub struct InMemSigner {
+    prng_seed: Option<u64>,
+    keys_generated: u64,
```

Comment:
I'm not sure I understand the purpose of this field.

Reply:
Me neither anymore.

Reply:
Actually, I do now. See https://github.com/linera-io/linera-protocol/pull/3696#discussion_r2035877539

Reply:
Possibly, but if we support keys' removal then no.